__tests__/index.test.js

```diff
@@ -129,3 +129,53 @@ describe("api/articles", () => {
       });
   });
 });
+describe("/api/articles/:article_id/comments", () => {
+   test("returns an empty array when the article id doesnt have any comments ", () => {
+     return request(app)
+       .get("/api/articles/2/comments")
+       .expect(200)
+       .then(({ body }) => {
+         const { comments } = body;
+         expect(comments).toHaveLength(0);
+         expect(comments).toEqual([]);
+       });
+   });
+   test("Responds with an array of comments from a specific article id", () => {
+    return request(app)
+      .get("/api/articles/9/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toHaveLength(2);
+        expect(comments).toBeSortedBy(comments.created_by);
```

Comment:
We need to check that the comments are sorted by the most recent ones, so could you just amend this please so it sorts by descending? If you check the jest sorted docs it should give an example of how to do this. 🙂